the-blue-alliance-ios/View Controllers/Match/MatchViewController.swift

```diff
@@ -39,18 +43,38 @@ class MatchViewController: MyTBAContainerViewController {
             tbaKit: tbaKit,
             userDefaults: userDefaults
         )
+        self.infoViewController.matchSummaryDelegate = self
```

Comment:
nit: can we do a newline after the init, and then set the delegate?  (we usually leave some space https://github.com/the-blue-alliance/the-blue-alliance-ios/blob/master/the-blue-alliance-ios/View%20Controllers/Events/EventsContainerViewController.swift#L31)

Also, we can drop the `self.` off of this and just set the delegate by doing `infoViewController.matchSummaryDelegate`